proxy/proxy.go

```diff
@@ -541,6 +541,9 @@ func (p *Proxy) applyFiltersToRequest(f []*routing.RouteFilter, ctx *context) []
 
 	var filters = make([]*routing.RouteFilter, 0, len(f))
 	for _, fi := range f {
+		if fi.Filter == nil {
```

Comment:
as explained in the other comment: I really prefer the noop filter way instead of this. OTOH, if we decide in the end to go with this way, then we need to add the nil check in the applyFiltersToResponse() function as well.

Reply:
(talking about this comment: https://github.com/zalando/skipper/pull/818/files#r228971780)

Reply:
solved in 7fea7d36ed038455ef267c668a987b0db3129e6d